src/ui/embeds/figma/figma-embed.module.scss

```diff
@@ -0,0 +1,5 @@
+.iframe {
+  border-radius: 6px;
+  border: 1px solid var(--border-color, #babec9);
```

Comment:
use the css vars with the `--bit-` prefix

Reply:
ok done